types/koa__router/koa__router-tests.ts

```diff
@@ -187,13 +187,13 @@ router4.post('/foo', emptyMiddleware, emptyMiddleware, emptyMiddleware, routeHan
 router4.get('name', '/foo', emptyMiddleware, emptyMiddleware, routeHandler4);
 router4.get('name', '/foo', emptyMiddleware, emptyMiddleware, emptyMiddleware, routeHandler4);
 
-const router5 = new Router();
+const router5 = new Router<any, {}>();
```

Comment:
Is this required now?

Reply:
Yes. This `router5` test relies on this behavior, so it can `register` two different kinds of middleware:

```js
const middleware1: Koa.Middleware<BlahInterface> = (ctx, next) => {
    ctx.state.blah = "blah";
};
const middleware2: Koa.Middleware<WoohInterface> = (ctx, next) => {
    ctx.state.wooh = "blah";
};

router5.register('/bar', ['GET', 'DELETE'], [middleware1, middleware2]);
```